src/olympia/addons/models.py

```diff
@@ -128,31 +128,15 @@ def clean_slug(instance, slug_field='slug'):
     return instance
 
 
-class AddonManager(ManagerBase):
-
-    def __init__(self, include_deleted=False, include_unlisted=False):
-        # DO NOT change the default value of include_deleted and
-        # include_unlisted unless you've read through the comment just above
-        # the Addon managers declaration/instantiation and understand the
-        # consequences.
-        ManagerBase.__init__(self)
-        self.include_deleted = include_deleted
-        self.include_unlisted = include_unlisted
-
-    def get_queryset(self):
-        qs = super(AddonManager, self).get_queryset()
-        if not self.include_deleted:
-            qs = qs.exclude(status=amo.STATUS_DELETED)
-        if not self.include_unlisted:
-            qs = qs.exclude(is_listed=False)
-        return qs.transform(Addon.transformer)
-
+class AddonQuerySet(caching.CachingQuerySet):
```

Comment:
What does breaking this out give us, out of interest?  (And are there other Managers that could benefit from it?)